metricq/timeseries/__init__.py

```diff
@@ -9,7 +9,7 @@
 """Type alias for strings that represent metric names
 """
 
-JsonDict = Dict[str, Any]
+JsonDict = dict[str, Any]
```

Comment:
May I ask, if that would be better a `Mapping`?

Reply:
No. Just to complicated to reason about the various usage of this type.